index.html

```diff
@@ -40,4 +41,143 @@ <h1 class="heading">
+            <li class="card-detail option"><img src="images/bullets.png" alt="bullet"> 2015</li>
+          </ul>
+
+          <p> A daily selection of privtely personalized reads; no accounts or sign-ups required. </p>
+          <div class="language1 inline">
+            <ul>
+              <li class="langu">html</li>
+              <li class="langu">css</li>
+              <li class="langu">javascript</li>
+            </ul>
+          </div>
+        </div>
+        <button type="button" class="btn-one">See project</button>
+      </div>
+      <div class="project-card">
+
+        <img src="images/photo3.png" alt="Third Card">
+        <h2 class="head">Tonic</h2>
+        <div class="card-detail-box">
+          <ul>
+            <li class="card-detail canopy">CANOPY</li>
+            <li class="card-detail option"><img src="images/bullets.png" alt="bullet"> Back End Dev</li>
+            <li class="card-detail option"><img src="images/bullets.png" alt="bullet"> 2015</li>
+          </ul>
+
+          <p> A daily selection of privtely personalized reads; no accounts or sign-ups required. </p>
+          <div class="language1 inline">
+            <ul>
+              <li class="langu">html</li>
+              <li class="langu">css</li>
+              <li class="langu">javascript</li>
+            </ul>
+          </div>
+        </div>
+        <button type="button" class="btn-one">See project</button>
+      </div>
+      <div class="project-card">
+        <img src="images/photo4.png" alt="Forth Card">
+        <h2 class="head">Multi-post Stories</h2>
+        <div class="card-detail-box">
+          <ul>
+            <li class="card-detail canopy">CANOPY</li>
+            <li class="card-detail option"><img src="images/bullets.png" alt="bullet"> Back End Dev</li>
+            <li class="card-detail option"><img src="images/bullets.png" alt="bullet"> 2015</li>
+          </ul>
+          <p> A daily selection of privtely personalized reads; no accounts or sign-ups required. </p>
+          <div class="language1 inline">
+            <ul>
+              <li class="langu">html</li>
+              <li class="langu">css</li>
+              <li class="langu">javascript</li>
+            </ul>
+          </div>
+        </div>
+        <button type="button" class="btn-one">See project</button>
+      </div>
+    </section>
+    <section class="section-about">
+        <div class="about-grid">
+          <div class="text">
+            <div class="headline-text">
+              <h1>About Myself</h1>
+
+              <p class="paragraph1">Hello I’m a software developer! I can help you build a product , feature or website
+                Look through some of my work and experience! If you like what you see and have a
+                project you need coded, don’t hestiate to contact me.</p>
+              <p class="lets-connect"> LET'S CONNECT</p>
```

Comment:
- [ ] This text here is a title and as a result, it would be better to use an `hx` tag like an `h2` tag here. 
- [ ] Also you have not applied the correct color here, it would be good if you could fix that.

Reply:
- [ ] [REQUIRED] Kindly ensure you use a `hx` tag like an `h2` tag as recommended by the previous reviewer for the portion you have the ''LET'S CONNECT"

Reply:
👍🏽 All good now.